incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,44 @@
+package test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	const (
+		hncYamlEnvVar = "HNC_REPAIR"
+	)
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+		// Remove all possible objections HNC might have to deleting a network. Make sure it 
+		// has cascading deletion so we can delete any of its subnamespace descendants, and 
+		// make sure that it's not a subnamespace itself so we can delete it directly.
+		for _, ns := range nses {
+			mustRun("kubectl hns set", ns, "-a")
+			tryRun("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+			mustRun("kubectl delete ns", ns)
+		}
+		hncRecoverPath := os.Getenv(hncYamlEnvVar)
+		if hncRecoverPath == ""{
+			Skip("Environment variable HNC_REPAIR not set. Skipping reocovering HNC.")
+		}
```

Comment:
Did you try this out? I would have thought that the `Skip()` function had to be part of the _test_ (the `It()` function), not the cleanup.

Reply:
When you've tested this, can you please update the commit comment as well to say that you tested it with and without HNC_REPAIR set?

Reply:
Yes tested it and in the log it says "[SKIPPING] in Spec Teardown (AfterEach)".
Added to the commit message